scripts/jest_teardown.js

```diff
@@ -0,0 +1,12 @@
+const { MeiliSearch } = require('meilisearch')
+
+const HOST = 'http://localhost:7700'
+const API_KEY = 'masterKey'
+
+afterAll(async () => {
+  const client = new MeiliSearch({ host: HOST, apiKey: API_KEY })
+  await client.deleteIndex('movies')
+  const task = await client.deleteIndex('games')
+
+  await client.waitForTask(task.taskUid)
```

Comment:
Don't you have a method that accepts more than one `uid`? (or I'm misleading because of other SDKs...)

Reply:
I don't have one in js :(